src/libs/protocols/src/trans.rs

```diff
@@ -97,6 +97,7 @@ impl From<oci::LinuxCapabilities> for grpc::LinuxCapabilities {
     }
 }
 
+#[cfg(any(target_os = "linux", target_os = "solaris"))]
```

Comment:
Do we have any link of the upstream issue for this?

We're not building for solaris, so I'd just mark "linux" as the target os.